ledger-core/v2/virtual/object/object.go

```diff
@@ -297,12 +311,15 @@ func (sm *SMObject) stepWaitState(ctx smachine.ExecutionContext) smachine.StateU
 func (sm *SMObject) stepGotState(ctx smachine.ExecutionContext) smachine.StateUpdate {
 	if sm.ActiveOrderedPendingCount > 0 {
 		sm.createWaitPendingOrderedSM(ctx)
+		sm.createWaitOrderedPendingTableSM(ctx)
 	} else {
 		ctx.ApplyAdjustment(sm.orderedPendingListFilledCtl.NewValue(true))
 	}
 
 	if sm.ActiveUnorderedPendingCount == 0 {
```

Comment:
can you make both this if's (if sm.ActiveOrderedPendingCount > 0 and sm.ActiveUnorderedPendingCount == 0) looks identically?